packages/headless/src/controllers/core/folded-result-list/headless-core-folded-result-list.ts

```diff
@@ -249,7 +255,9 @@ export function buildCoreFoldedResultList(
 
 function loadFoldingReducer(
   engine: CoreEngine
-): engine is CoreEngine<SearchSection & ConfigurationSection & FoldingSection> {
+): engine is CoreEngine<
```

Comment:
Should not change anything, but there was a missing section in the type predicate.